cli-feed-v3-2.json

```diff
@@ -17,23 +17,23 @@
       "hidden": true
     },
     "v2": {
-      "release": "2.58.0",
+      "release": "2.59.0",

```

Comment:
Direct release to prod? It this intentional

Reply:
Following the instructions which say `update GA, preview and pre-release` tags to point to new release after successful OGF, yeah. Did things change ?

![image](https://user-images.githubusercontent.com/26885664/101544733-0a25f900-395b-11eb-87f2-1d3903f49170.png)

Reply:
was just making sure this was on purpose